notebooks/view_comments.ipynb

```diff
@@ -214,108 +214,230 @@
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.franchise_name IS 'Franchise Name'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.project_county IS 'County where project occurs'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.project_state IS 'State where project occurs'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.sba_district_office IS 'SBA district office for the loan'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.congressional_district IS 'Congressional district office, within each state, for the borrower address'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.business_type IS 'Borrower Business Type - Individual, Partnership, or Corporation'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.loan_status IS 'Status of the loan. PIF=Paid in Full; EXEMPT=Not Disclosed, treat like Unknown; CHGOFF=Charged Off; NOT FUNDED=Not yet disbursed; CANCLED=Cancelled'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.chargeoff_date IS 'Date SBA charged off loan'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.gross_chargeoff_amount IS 'Total loan balance charged off (includes guaranteed and non-guaranteed portion of loan)'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.jobs_supported IS 'Total Jobs Created + Jobs Retained as reported by lender on SBA Loan Application.  SBA does not review, audit, or validate these numbers - they are simply self-reported, good faith estimates by the lender.'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.cdc_name IS 'Community Development Corporation name'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.cdc_street IS 'Community Development Corporation street address'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.cdc_city IS 'Community Development Corporation city'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.cdc_state IS 'Community Development Corporation state'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.cdc_zip IS 'Community Development Corporation zip code'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.third_party_lender_name IS 'Third Party Lender name'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.third_party_lender_city IS 'Third Party Lender city'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.third_party_lender_state IS 'Third Party Lender state'\n",
+    ";COMMENT ON COLUMN data_ingest.sba__foia_504_1991_present.third_party_dollars IS 'Third Party loan amount'\n",
     "\n",
-    "In future, this should be made iterable, postgres has a limitation to selecting the table comment through a function"
+    "\n",
+    "\n",
+    ";\n",
+    "--SQL Alchemy throws an error unless rows are returned. This works around the issue\n",
```

Comment:
Ah! Yes, using `pd.read_sql` is not really the right way to execute general SQL statements. The point of `pd.read_sql` is to return a DataFrame object, so generally is used with `select` SQL statements. To execute SQL in `sqlalchemy`.

To execute SQL in `sqlalchemy` you can use `connection.execute`. See the docs for an example: http://docs.sqlalchemy.org/en/latest/core/connections.html.

In any case it's better that you added the comment stuff in the pipeline code. That way everything touching the database is in one place (and it's easy to recreate the state of the database at any point in time.